docs/getting_started/install_on_ibexa_engage.md

```diff
@@ -0,0 +1,52 @@
+---
+description: Install and configure Ibexa Engage.
+edition: experience
+---
+
+# Install on Ibexa Engage
+
+[[= product_name_engage =]] is a data collection tool. It enables you to engage your audiences by using the [Qualifio](https://qualifio.com/) tools.
+You can use interactive content to gather valuable data, for example, customer personal data or recent orders list, and create connections.
+
+## Prepare configuration files
+
+First, run the `composer ibexa:setup` command:
+
+``` bash
+composer require ibexa/connector-qualifio
+```
+
+This command adds to your project configuration files required for using [[= product_name_engage =]].
+
+## Create an account
+
```

Comment:
Ibexa will create and provide the account/users (client will be invited during the provisioning)